src/env/test/mod.rs

```diff
@@ -1,20 +1,42 @@
 use self::upgrade_storage::BufferUpgradeStorage;
-use crate::api::customization::{CustomizationImpl, DEFAULT_CUSTOMIZATION};
+use crate::api::customization::DEFAULT_CUSTOMIZATION;
 use crate::api::firmware_protection::FirmwareProtection;
 use crate::ctap::status_code::Ctap2StatusCode;
 use crate::ctap::Channel;
 use crate::env::{Env, UserPresence};
-use crypto::rng256::ThreadRng256;
+use crypto::rng256::Rng256;
+use customization::TestCustomization;
 use persistent_store::{BufferOptions, BufferStorage, Store};
+use rand::rngs::StdRng;
+use rand::{Rng, SeedableRng};
 
+mod customization;
 mod upgrade_storage;
 
 pub struct TestEnv {
-    rng: ThreadRng256,
+    rng: TestRng256,
     user_presence: TestUserPresence,
     store: Store<BufferStorage>,
     upgrade_storage: Option<BufferUpgradeStorage>,
-    customization: CustomizationImpl,
+    customization: TestCustomization,
+}
+
+pub struct TestRng256 {
+    rng: StdRng,
+}
+
+impl TestRng256 {
+    pub fn seed_rng_from_u64(&mut self, state: u64) {
```

Comment:
I missed the fact that this was `seed_rng_from_u64`. I would remove the `_rng` in your next PR since it's redundant.